snow/engine/snowman/block/block_context_vm.go

```diff
@@ -21,7 +21,9 @@ type Context struct {
 	//
 	// Because PreForkBlocks and PostForkOptions do not verify their execution
 	// against the P-chain's state, this context is undefined for those blocks.
-	PChainHeight uint64
+	PChainHeight uint64 `canoto:"fint64,1" json:"pChainHeight"`
```

Comment:
I don't think this should be a `fint64` - I think this should just be an `int`. (The `PChainHeight` is not going to be > 2^56 for a long time). Even if this was `fint32` I think `int` is more efficient for the foreseeable future.